applications/openshift/api-server/api_server_api_priority_flowschema_catch_all/rule.yml

```diff
@@ -26,6 +26,10 @@ rationale: |-
 identifiers:
   cce@ocp4: CCE-83522-3
 
+platforms:
+  - ocp4.8
+  - ocp4.9
+  - ocp4.10
```

Comment:
I worry that the list of platform will keep growing bigger and bigger.

Do you think it would make sense to have platforms named `flowschemas_v1alpha1` and `flowschemas_v1beta1`? And they would know and check the OCP version that use each flowschemas.

Another approach: If the platform check is written to check for version ranges, instead of individual versions, maintenance of the platform would happen for each change in flowschemas version opposed to every ocp release. For example, `flowschemas_v1beta1` is for ocp 4.8 or higher.
But this would be a problem if it is expected that older content versions work on newer ocp. And I think it would not be a problem for newer content version to work on older releases.